package.json

```diff
@@ -17,27 +17,28 @@
   "license": "Apache-2.0",
   "private": true,
   "devDependencies": {
-    "babel-core": "^6.23.1",
-    "babel-loader": "^7.1.2",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-react": "^6.23.0",
-    "babel-preset-stage-2": "^6.24.1",
-    "concurrently": "^3.5.1",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
-    "enzyme-to-json": "^3.3.1",
-    "jest": "^22.0.0",
-    "jest-fetch-mock": "^1.4.2",
-    "jest-image-snapshot": "^2.3.0",
-    "jest-puppeteer-preset": "^2.0.1",
-    "node-fetch": "^2.1.1",
+    "@babel/core": "^7.7.7",
```

Comment:
I believe `babel-preset-amex` includes `@babel/core` already. Do we need to include it here?

Reply:
We don't want to add more deps than necessary.

Reply:
Updated in 52e7525